examples/coexist_klog_v1_using_logr/coexist_klog_v1.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"flag"
+	"github.com/go-logr/logr"
+
+	"k8s.io/klog"
+	klogv2 "k8s.io/klog/v2"
+)
+
+type klogLogger struct{}
+
+func (kw klogLogger) Info(msg string, keysAndValues ...interface{}) {
+	klog.InfoDepth(5, msg)
+}
+
+func (kw klogLogger) Enabled() bool {
+	return true
+}
+
+func (kw klogLogger) Error(err error, msg string, keysAndValues ...interface{}) {
+	klog.ErrorDepth(5, msg)
+}
+
+func (kw klogLogger) V(level int) logr.InfoLogger {
+	return kw
```

Comment:
this drops verbosity info, right?

Reply:
yea, i like the SetOutput solution better at this point